src/components/Sidebar.tsx

```diff
@@ -0,0 +1,35 @@
+import { Bars3Icon, XMarkIcon } from "@heroicons/react/24/solid";
+import { useState } from "react";
+import Settings from "./Settings";
+
+const Sidebar = () => {
+  const [showSidebar, setShowSidebar] = useState(false);
+
+  return (
+    <>
+      {showSidebar ? (
+        <button
+          aria-label="Settings Button"
```

Comment:
Nit: "Settings Button" would be fine for an `alt` label but `aria-label` is more for the functionality of this button, so you can say:

```suggestion
          aria-label="Open settings menu"
```